resources/energyplus/ProposedEnergy+.idd

```diff
@@ -533,35 +559,52 @@ ShadowCalculation,
        \min-fields 2
        \memo This object is used to control details of the solar, shading, and daylighting models
        \extensible:1
-  A1 , \field Calculation Method
-       \note choose calculation method. note that TimestepFrequency is only needed for certain cases
+  A1 , \field Shading Calculation Method
+       \note Select between CPU-based polygon clipping method, the GPU-based pixel counting method,
+       \note or importing from external shading data.
+       \note If PixelCounting is selected and GPU hardware (or GPU emulation) is not available, a warning will be
+       \note displayed and EnergyPlus will revert to PolygonClipping.
+       \note If Scheduled is chosen, the External Shading Fraction Schedule Name is required
+       \note in SurfaceProperty:LocalEnvironment.
+       \note If Imported is chosen, the Schedule:File:Shading object is required.
+       \type choice
+       \key PolygonClipping
+       \key PixelCounting
+       \key Scheduled
+       \key Imported
+       \default PolygonClipping
+  A2 , \field Shading Calculation Update Frequency Method
+       \note choose calculation frequency method. note that Timestep is only needed for certain cases
        \note and can increase execution time significantly.
        \type choice
-       \key AverageOverDaysInFrequency
-       \key TimestepFrequency
-       \default AverageOverDaysInFrequency
-  N1 , \field Calculation Frequency
+       \key Periodic
+       \key Timestep
+       \default Periodic
+  N1 , \field Shading Calculation Update Frequency
```

Comment:
New field to be added, and one field renamed with keys renamed + one field rename. Will need VT.

Given the names, I think we should just break API otherwise it's going to be very confusing, and it's 3.x so that's kinda ok.

* <OLD> => <NEW>
* <not_existing> =>  `Shading Calculation Method`
* `Calculation Method` => `Shading Calculation Update Frequency Method`

@kbenne @tijcolem thoughts?

Reply:
Moved to its own issue at https://github.com/NREL/OpenStudio/issues/3908